docs/modules/ROOT/pages/configuration-store.adoc

```diff
@@ -0,0 +1,172 @@
+= Universal Configuration Store
+
+== Overview
+The Universal Configuration Store is a tool that enables the various configurations for a project to be centrally defined and managed. 
+It then provides a standardized way of accessing them, allowing the environment specific configurations to be dynamically provided to 
+their respective resources within the project at runtime.
+
+=== Setup
+To setup the configuration store, one must first create properties files for storing the configurations. Each individual configuration is 
+represented as a property key value pair within the file. These files must then be made accessible to the configuration store kubernetes 
+resource for ingestion. See the applicable sections below for more details on defining configurations as properties, creating environment 
```

Comment:
A: Any time we say "See XYZ..." we should just link the section we're talking about.  In-window links for any internal navigation and new-window links for any external navigation.  I'd also suggest where we can working the link more organically into a sentence rather than outright saying "See XYZ". E.g. The first sentence could read _"...must first [define the configurations] through properties files."_ where _define the configurations_ links to the relevant section.

Reply:
Update hyperlinks to flow with sentence